Functions.Templates/Templates/EventGridTrigger-PowerShell/run.ps1

```diff
@@ -0,0 +1,3 @@
+param($eventGridEvent, $TriggerMetadata)
+
+$eventGridEvent | Out-String | Write-Host

```

Comment:
NIT: maybe add a comment explaining that if you want to log an object, pass it through `Out-String` so users understand why this line exists

Reply:
added